server/app/com/xsn/explorer/data/anorm/dao/BlockPostgresDAO.scala

```diff
@@ -307,6 +307,112 @@ class BlockPostgresDAO @Inject()(
     }
   }
 
+  def getBlocks(limit: Limit, orderingCondition: OrderingCondition)(implicit conn: Connection): List[BlockInfo] = {
+    val order = toSQL(orderingCondition)
+
+    SQL(
+      s"""
+        |SELECT blk.*, COALESCE(tx.count, 0) transactions 
```

Comment:
today I learned that this syntax is accepted by postgres (`value alias` instead of `value as alias`)